### PR TITLE
Tests: Remove Cypress

### PR DESCRIPTION
**Pull request by batbattur**
## Summary
Removed all of the Cypress-related files including:
- test-helpers
- cypress and cypress-related packages/libraries. Used `pnpm remove -D <package-name>`.
- all cypress tests and configs
- cypress workflow

## QA notes
1. Go to the collections page (either `/Parts` or `/Tools`) and click the pagination button for the next search result. 
Then make sure it scrolls smoothly (same as on master).
2. Run `git grep -i cypress` and make sure no actual code shows up in the result. 
Note: there should be only one cypress comment in `playwright/utils.ts`. 

closes: #1239 #1111

### CODE REVIEW

**Comment by deltuh-vee** [comment]
QA 🎬 
* Autoscrolling is still smooth on collection pages
* No results appear from `git grep -i cypress` except for the one in `playwright/utils.ts`

File: package.json [hunk 1]
```diff
@@ -21,8 +21,6 @@
       "strapi": "cd backend && pnpm run strapi",
       "test": "pnpm run -r test",
       "transform-svg": "cd frontend && pnpm run transform-svg",
-      "cypress:open": "cd frontend && pnpm run cypress:open",
```
Comment (ardelato) [inline package.json:24]: Oooh shoot I didn't know we had these scripts in the root `package.json`. We should probably add these scripts for `playwright`

**Review by ardelato — APPROVED** [review]
CR ⚡ There were a lot more references to `Cypress` then I thought. Nice job covering all bases @batbattur 🙌

File: package.json (hunk 1 above)
Comment (batbattur) [reply package.json:24]: Yes! I think we should add them back and also update the `Test` section in read me.
Let's do it in a follow-up issue 👍

Comment (batbattur) [reply package.json:24]: Created a follow-up issue: https://github.com/iFixit/react-commerce/issues/1253.

**Pull request merged**